public/employers/united-airlines.yml

```diff
@@ -50,3 +50,18 @@ citations:
     sources:
       - source: United Hub
         link: https://web.archive.org/web/20200427070524/https://hub.united.com/united-coronavirus-covid19-safety-update-2645397564.html # Unknown date.
+  - summary: United announced a social-distancing measure that would block middle seats from being occupied, but later clarified that this only affected *seat selection*, not *capacity*, so a fully-booked flight would have no social distancing measures.
+    positivity: 0
```

Comment:
I feel like this should be -1 as it is misleading to customers and could endanger employees as well.